_quarto.yml

```diff
@@ -50,7 +58,10 @@ website:
           - guidance/03-responsibilities/pages/09-commissioner-responsibilities.qmd
           - guidance/03-responsibilities/pages/10-analyst-responsibilities.qmd
           - guidance/03-responsibilities/pages/11-critical-friend.qmd
-      
+    page-footer:
+      left:
+        - text: "Accessibility"
+          href: accessibility.qmd
```

Comment:
This is a markdown version of the official accessibility statement, since we can avoid having to add that to the public website by adding here instead.